api/models.py

```diff
@@ -1,3 +1,25 @@
 from django.db import models
+from django.utils import timezone
+from django.contrib.auth.models import User
 
-# Create your models here.
+ACTIVITY_LEVEL = (
+			("SC", "Sleepy_Cat"),
+			("LC", "Lazy_Cat"),
+			("AC", "Active_Cat"),
+			("MC", "Mentor_Cat"),
+	)
```

Comment:
Quotes issue is a whole other issue. The whole programming world is talking and arguing about the single vs. double quotes. 
The idea is to choose one and then stick to them throughout the whole project. I am partial to the single quotes as I find them helpful with the HTML notation which often uses double quotes, but you guys should agree on one and adopt it throughout the project.

We proper programmers now, already talking about single vs. double quotes. :)